### DISCUSSION TITLE
1panel商店没有halo

### DISCUSSION THREAD

**Discussion by wjlin0** [1Panel-dev/1Panel#7580] [General]
为什么安装后没有halo 还有一些也没有
<img width="1482" alt="image" src="https://github.com/user-attachments/assets/ae6d26d1-3183-435e-b0ed-5fb39175de91" />

**Comment by zhengkunwang223** [comment]
![image](https://github.com/user-attachments/assets/af31fb73-58ae-40e8-8c28-6d9fa6c92e42)
使用此脚本安装的为海外版 海外版应用商店中应用会少一些

https://1panel.cn/docs/installation/online_installation/ 可以在这安装国内版
